tests/test_admin.py

```diff
@@ -647,3 +647,128 @@ def test_cms_page_toolbar_pages_link_doesnt_filter_the_page_list_by_page_id(self
 
         self.assertSetEqual(set(response.context["cl"].queryset), set(pages))
         self.assertSetEqual(set(response_with_page_id.context["cl"].queryset), set(pages))
+
+
+class PageAdminCsvExportFileTestCase(CMSTestCase):
+    def setUp(self):
+        # CSV Headings: 0 -> Title, 1 -> Content Type, 2 -> Expiry Date, 3 -> Version State, 4 -> Author, 5 -> Url
```

Comment:
Would you be able to swap this out for a more detailed comment, or remove it?

Reply:
Removed the comment